src/app/app.component.ts

```diff
@@ -17,7 +16,7 @@ export class AppComponent {
     this.currentSearchData$ = this.store.select('CurrentSearchReducer');
```

Comment:
As far as I know, _select_ is used to retrieve state, not reducers, so I cannot understand why you use keys referencing _reducers_.